### PR TITLE
Example of registry arguments

### PR DESCRIPTION
**Pull request by jeffb-stell**
Gives a quick example on accepting/passing arguments into a registry method

### CODE REVIEW

File: docs/building-blocks.md [hunk 1]
```diff
@@ -309,6 +309,28 @@ SparkleFormation.new(:instance_stack) do
 end
 ~~~
 
+Additional parameters can be passed into registry methods
+
+~~~ruby
+SfnRegisty.register(:add_foo_bar) do |args={}|
```
Comment (luckymike) [inline docs/building-blocks.md]: This name needs to match what's called in the template (or vice-versa).

**Comment by luckymike** [comment]
Thanks for the PR @jeffb-stell, this should definitely be documented.

A couple things that need adjustment before we can accept:

1. The PR should target `develop`
2. The Registry example you made has a different name than what's called in the template. See the line note.
3. The foo_bar example demonstrates the ability to work with args in the registry via ruby operators, but doesn't line up with the parameter example. For clarity, this should probably be adjusted to show an example of passing a default instance type, which could be done with arguments for `:class` (t2, m3, m4, etc.)  & `:size` (medium, large, xlarge, etc.), which you combine in the registry.

**Comment by chrisroberts** [comment]
Closed in favor of #153. Thanks!

File: docs/building-blocks.md (hunk 1 above)
Comment (jeffb-stell) [inline docs/building-blocks.md]: doh, apparently I was copying/pasting out of my code. Thanks for the correction (although given #153 I'll close out the branch unfixed :)

**Pull request closed without merging**